mobile/src/main/java/com/truckmuncher/app/customer/TruckDetailsActivity.java

```diff
@@ -95,14 +95,7 @@ protected void onCreate(Bundle savedInstanceState) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
-                Intent parent = NavUtils.getParentActivityIntent(this);
-                if (NavUtils.shouldUpRecreateTask(this, parent)) {
-                    NavUtils.navigateUpTo(this, parent);
-                } else {
-                    TaskStackBuilder.create(this)
-                            .addParentStack(this)
-                            .startActivities();
-                }
```

Comment:
I don't think the deep linking is working properly for me. I tried clicking a link in a Hangouts chat and the truck's menu shows up, but clicking the back button takes me back to Hangouts. I tried looking at my open apps after clicking the link and TruckMuncher wasn't open at all. The truck details content was inside the Hangout app.